homeassistant/package_constraints.txt

```diff
@@ -99,3 +99,7 @@ multidict>=6.0.2
 # Required for compatibility with point integration - ensure_active_token
 # https://github.com/home-assistant/core/pull/68176
 authlib<1.0
+
+# Required for compatibility with typer, used by pyunifiprotect integration
+# https://github.com/tiangolo/typer/pull/375
+click<=8.0.4"
```

Comment:
We should fix the version pinning (it doesn't break CI though - for whatever reason): https://github.com/home-assistant/core/pull/68808#discussion_r837219963

Reply:
#68833 - I don't know if it is needed for 2022.3.8 or just for dev

Reply:
https://github.com/home-assistant/core/pull/68833

Reply:
CI seems to work, up to you

Reply:
#68833 has been merged - it can be cherry-picked